doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -113,18 +140,26 @@ term consists of the successive application of its coercions.
 Declaring Coercions
 -------------------------
 
-.. cmd:: Coercion @reference : @class >-> @class
-         Coercion @ident {? @univ_decl } @def_body
+.. cmd:: Coercion @reference {? : @class >-> @class }
+         Coercion @ident_decl @def_body
 
   The first form declares the construction denoted by :token:`reference` as a coercion between
-  the two given classes.  The second form defines :token:`ident`
-  just like :cmd:`Definition` :n:`@ident {? @univ_decl } @def_body`
-  and then declares :token:`ident` as a coercion between it source and its target.
+  the two given classes.  The second form defines :token:`ident_decl`
+  just like :cmd:`Definition` :n:`@ident_decl @def_body`
+  and then declares :token:`ident_decl` as a coercion between it source and its target.
   Both forms support the :attr:`local` attribute, which makes the coercion local to the current section.
 
-  This command supports the :attr:`local` and :attr:`global` attributes
-  (described :ref:`here <set_unset_scope_qualifiers>`) as well as the
-  :attr:`nonuniform` attribute.
+  :n:`{? : @class >-> @class }`
+    The source and target classes of the coercion.
+    If unspecified, :n:`@reference` must already be a coercion, which
+    enables modifying the :attr:`reversible` attribute of :n:`@reference`.
+    See the :ref:`example <example-reversible-coercion-attribute>` below.
+
+  .. attr:: reversible{? = {| yes | no } }
+     :name: reversible
+
+     Allows the coercion to be used as a reversible coercion.  The default is
+     :n:`no` except for fields specified using :g:`:>` in :cmd:`Record`.
```

Comment:
"The default is no" is likely to mislead the uninformed reader who may think that `reversible` means `reversible = no` (when in fact it means `reversible = yes`).

Instead of writing "Allows...", I recommend beginning by ``This :term:`attribute` allows`` (and same thing for the `nonuniform` attribute below BTW) because this gives an opportunity to the user who doesn't know how attribute work to click the link and read about it.

Reply:
done